pom.xml

```diff
@@ -28,7 +28,7 @@
     <changelist>-SNAPSHOT</changelist>
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
     <apacheds.version>2.0.0.AM25</apacheds.version>
-    <jenkins.version>2.235.3</jenkins.version>
+    <jenkins.version>2.257-rc30526.4bb45a702b8d</jenkins.version> <!-- TODO https://github.com/jenkinsci/jenkins/pull/4848 -->
```

Comment:
To be released in the next weekly. Will this be ready to review and merge afterward?

Reply:
It has already been reviewed (more reviews welcome!) and yes it would be ready to merge when the dep is on 2.266.